tests/integration_test.py

```diff
@@ -322,9 +354,11 @@ def test_valid_no_config_specified(self):
         self.assertEqual(container_log_config['Config'], {})
 
 
-@unittest.skipIf(not EXEC_DRIVER_IS_NATIVE, 'Exec driver not native')
 class TestCreateContainerReadOnlyFs(BaseTestCase):
     def runTest(self):
+        if not exec_driver_is_native():
+            pytest.skip('Exec driver not native')
```

Comment:
Is there a reason we don't use the `@pytest.mark.skipif` decorator here as well?